web/src/main/java/org/cbioportal/web/util/StudyViewFilterApplier.java

```diff
@@ -272,8 +271,7 @@ public SampleIdentifier apply(Sample sample) {
     }
 
     private List<SampleIdentifier> filterMutatedGenes(List<GeneFilter> mutatedGenefilters,
-            Map<String, MolecularProfile> molecularProfileMap, MolecularAlterationType molecularAlterationFilterType,
-            List<SampleIdentifier> sampleIdentifiers) {
+            Map<String, MolecularProfile> molecularProfileMap, List<SampleIdentifier> sampleIdentifiers) {
```

Comment:
So what was `molecularAlterationFilterType` doing here? It looks like it was doing absolutely nothing - is that correct?

Reply:
It was useful before when both mutation and fusion/SV data requests are handled together. I forgot to clean it up once I introduced new service for fetching/filtering fusion/SV data